docs/user-guide/administration/manage-etcd.md

```diff
@@ -245,3 +300,170 @@ spec:
 ```
 
 Restart GreptimeDB Metasrv to complete etcd restore.
+
+## Monitoring
+
+- Prometheus Operator installed (e.g. via [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)).
+- podmonitor CRD installed (automatically installed with Prometheus Operator).
+
+Add the following to your `etcd-monitoring.yaml` to enable monitoring:
+
+```yaml
+replicaCount: 3
+
+auth:
+  rbac:
+    create: false
+  token:
+    enabled: false
+
+persistence:
+  storageClass: null
+  size: 8Gi
+
+resources:
+  limits:
+    cpu: '2'
+    memory: 2Gi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+autoCompactionMode: "periodic"
+autoCompactionRetention: "1h"
+
+extraEnvVars:
+  - name: ETCD_QUOTA_BACKEND_BYTES
+    value: "8589934592"
+  - name: ETCD_ELECTION_TIMEOUT
+    value: "2000"
+
+# Monitoring settings
+metrics:
+  enabled: true
+  podMonitor:
+    enabled: true
+    namespace: etcd-cluster
+    interval: 10s
+    scrapeTimeout: 10s
+    additionalLabels:
+      release: prometheus
+```
+
+Deploy etcd with Monitoring:
+
+```bash
+helm upgrade --install etcd \
+  oci://registry-1.docker.io/bitnamicharts/etcd \
+  --create-namespace \
+  -n etcd-cluster \
+  --values etcd-monitoring.yaml
+```
+
+### Grafana dashboard
+
+Use the [ETCD Cluster Overview dashboard](https://grafana.com/grafana/dashboards/15308-etcd-cluster-overview/) (ID: 15308) for monitoring key metrics.
+
+1. Log in your Grafana.
+2. Navigate to Dashboards -> New -> Import.
+3. Enter Dashboard ID: 15308, select the data source and load.
+
+![ETCD Cluster Overview dashboard](/etcd-cluster-overview-dashboard.png)
+
+## Defrag
+
+ETCD uses a multi-version concurrency control (MVCC) mechanism that stores multiple versions of KV. Over time, as data is updated and deleted, the backend database can become fragmented, leading to increased storage usage and reduced performance. Defragmentation is the process of compacting this storage to reclaim space and improve performance.
```

Comment:
Defragmentation is very dangerous since it will block reading and writing, etc. We should add the warning note.

Reply:
Already add warning in defrag step.